### PR TITLE
New in Chrome 89

### PR DESCRIPTION
**Pull request by petele**
Changes proposed in this pull request:
- Written post for New in Chrome 89

### CODE REVIEW

File: site/en/blog/new-in-chrome-89/index.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+I'm [Pete LePage](https://twitter.com/petele), working, and shooting
+from home, let's dive in and see what's new for developers in Chrome 89!
+
+## WebHID, WebNFC, and Web Serial {: #hardware }
+
+I'm really excited about WebHID, WebNFC, and Web Serial. They open up new
+scenarios for users that were never possible before, interacting with real
+world hardware.
+
+It allows makers to connect to fun, quirky hardware, video conferencing apps
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-89/index.md:43]:
```suggestion
They allows makers to connect to fun, quirky hardware, video conferencing apps
```

Comment (jpmedley) [reply site/en/blog/new-in-chrome-89/index.md:43]: I realize this is what you said in the video, but referring to a list of features with a singular pronoun has a high chance causing confusion.

File: site/en/blog/new-in-chrome-89/index.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+of other use cases.
+
+{% Video className="float-right", autoplay="true", muted="true", loop="true", src="video/0g2WvpbGRGdVs0aAPc6ObG7gkud2/g8VCbIMljnxuemDFxl99.mp4" %}
+
+Using Web Serial, and [about 60 lines of code][pico-code],
+[@AndreBan][andre-tweet] created a [page][andre-page] that can interact with
+the MicroPython REPL on a [Raspberry Pi Pico][rpi-pico]. Web Serial is also
+used by Espruino in their [web based IDE][espruino-ide].
+
+At CDS last year, Francois wrote a fun, memory-style game using Web NFC.
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-89/index.md:54]: Actually it was 2019.

File: site/en/blog/new-in-chrome-89/index.md [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+<div style="clear:both;"></div>
+
+{% Img src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/VAuLti7I9ZQCwLUEWF5Y.png", className="float-right", alt="StreamDeck with Daft Punk Drum Pad", width="800", height="536" %}
+
+And my favorite, [@bramus][bramus-tweet] used WebHID to connect to a
+StreamDeck, building a [Daft Punk drum pad][bramus-demo]. If you don't have
+a StreamDeck, check out his [demo video][bramus-demo-video] on YouTube, and
+check out the [code on GitHub][bramus-code].
+
+Whether it's your site, that interacts with your hardware, or your hardware
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-89/index.md:66]:
```suggestion
Whether it's your site that interacts with your hardware, or your hardware
```

File: site/en/blog/new-in-chrome-89/index.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+
+You can learn more about some of the devices you can connect to at
+[web.dev/devices](https://web.dev/devices/), or check out the getting started
+guides for [WebHID](https://web.dev/hid/), [WebNFC](https://web.dev/nfc/),
+and [Web Serial](https://web.dev/serial/).
+
+## PWA installability criteria changes {: #installability }
+
+Offline support has been a key part of the Progressive Web App criteria for
+installability since the beginning. Like any other installed app, users
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-89/index.md:78]:
```suggestion
installability since the beginning. As with other installed app, users
```

Comment (jpmedley) [reply site/en/blog/new-in-chrome-89/index.md:78]: I know this looks like the old like/as confusion. Actually what you have is a dangling modifier that makes it sound like a user is an installed app.

File: site/en/blog/new-in-chrome-89/index.md [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+expect it to work reliably, it should be fast, and, they should never see
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-89/index.md:79]:
```suggestion
expect it to work reliably. It should be fast, and, they should never see
```

File: site/en/blog/new-in-chrome-89/index.md [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+the offline dino!
+
+Later this year, we plan to close a loophole that allowed a few sites to
+pass the installability criteria, without an offline experience. **If your
+PWA already has an offline experience, you're all set, there's no
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-89/index.md:84]:
```suggestion
PWA already has an offline experience, you're all set. There's no
```

File: site/en/blog/new-in-chrome-89/index.md [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+action required, but, if you don't, it's time to add one!**
+
+Starting in Chrome 89, if your PWA doesn't provide a valid response when
+offline, you'll see a warning DevTools under the Issues tab, and Lighthouse
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-89/index.md:88]:
```suggestion
offline, you'll see a warning in DevTools under the Issues tab, and Lighthouse
```

File: site/en/blog/new-in-chrome-89/index.md [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+## Web Share and Web Share Target for Desktop {: #webshare }
+
+{% Video className="float-right", autoplay="true", muted="true", loop="true", src=["video/0g2WvpbGRGdVs0aAPc6ObG7gkud2/0vmJ4uVRmhkufMBsNrEO.webm","video/0g2WvpbGRGdVs0aAPc6ObG7gkud2/8rZz40MCN1qTifH2FHqG.mp4"] %}
+
+If your site allows users to create, edit, or interact with files, you should
+be using the Web Share and Web Share Target APIs. These APIs have been
+available on mobile for some time but are now supported on Chrome OS and
+Windows.
+
+Web Share makes it possible for users to send files, or data to other
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-89/index.md:128]:
```suggestion
Web Share makes it possible for users to send files or data to other
```

File: site/en/blog/new-in-chrome-89/index.md [hunk 9]
```diff
@@ -0,0 +1,223 @@
...
+    await navigator.share({title, text, url});
+    return true;
+  } catch (ex) {
+    console.error('Share failed', ex);
+    return false;
+  }
+}
+```
+
+To register as a target, so other apps can share files or data with you,
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-89/index.md:146]:
```suggestion
To register as a target so other apps can share files or data with you,
```

File: site/en/blog/new-in-chrome-89/index.md [hunk 10]
```diff
@@ -0,0 +1,223 @@
...
+* [Chromium source repository change list](https://chromium.googlesource.com/chromium/src/+log/88.0.4324.98..89.0.4389.75)
+
+## Subscribe
+
+Want to stay up to date with our videos, then [subscribe](https://goo.gl/6FP1a5)
+to our [Chrome Developers YouTube channel](https://www.youtube.com/user/ChromeDevelopers/),
+and you'll get an email notification whenever we launch a new video.
+
+I'm Pete LePage, and as soon as Chrome 90 is released, I'll be right here to
+tell you -- what's new in Chrome!
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-89/index.md:206]:
```suggestion
tell you what's new in Chrome!
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/blog/new-in-chrome-89/index.md (hunk 2 above)
Comment (petele) [reply site/en/blog/new-in-chrome-89/index.md:54]:
```suggestion
At CDS 2019, Francois wrote a fun, memory-style game using Web NFC.
```

**Review by jpmedley — APPROVED** [review]

**Pull request merged**